packages/RatPack.package/RatPackHowTo.class/class/page3.st

```diff
@@ -0,0 +1,14 @@
+pages
+page3
+	^HelpTopic 
+		title: '3. Setting up an Object'
+		contents: 'Setting up an Object
+
+	When in workspace:
+
+		app persistObjectBy: aValidJsonString from: ModelClass.
+	
+	When in RPControlPanel:
+
+		- Ctrl-o to add an object
+		- Ctrl-i to remove an object'
```

Comment:
I'd add:
RatPack will also set up a REST-API for all registered models when you start the server. This can also be used to create new objects in the database. You can access it at server.com/api/.